pkg/image/online.go

```diff
@@ -45,6 +45,12 @@ type UpdateInstallationImagesOptions struct {
 	DockerHubRegistryCreds registry.Credentials
 }
 
+type UpdateInstallationAirgapArtifactsOptions struct {
```

Comment:
this is long but 🤷‍♂️ 
```suggestion
type UpdateInstallationEmbeddedClusterArtifactsOptions struct {
```